pkgs/by-name/en/ente-auth/update.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gojq nix-prefetch-github common-updater-scripts
+
+set -eou pipefail
+pkg_dir="$(dirname "$0")"
+
+version="$(curl --silent "https://api.github.com/repos/ente-io/ente/releases" | gojq 'map(select(.draft == false and .prerelease == false and (.tag_name | startswith("auth-v")))) | first | .tag_name' --raw-output)"
```

Comment:
This should optionally take a GITHUB_TOKEN https://github.com/NixOS/nixpkgs/blob/21576aba43e604a95cc45c25248f39f3989ebe36/pkgs/applications/finance/denaro/update.sh#L7